fastlane/Fastfile

```
@@ -16,23 +16,35 @@ platform :android do
     gradle(
       task: "bundleRelease",
     )
-    upload_to_play_store(track: 'internal')
+    upload_to_play_store(
+      track: 'internal',
+      aab: 'app/build/outputs/bundle/release/app-release.aab',
+      skip_upload_images: true,
```

RuboCop (1.73)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)